gadgets/trace_dns/program.bpf.c

```diff
@@ -153,253 +188,146 @@ struct {
 	__uint(max_entries, 1024);
 } query_map SEC(".maps");
```

Comment:
I am wondering whether to implement the latency field in Wasm so the `query_map` bpf map would become a Go LRU map and it could be cleanup automatically.

At the moment, it seems easy to generate traffic in a way where `query_map` becomes full and the gadget stops working.

Reply:
I think I agree with you. Is it fine if we make this change in another PR? (We'll need to think if we want to make the same change on the built-in version)